mifospay/src/main/java/org/mifos/mobilewallet/mifospay/qr/ui/ShowQrActivity.java

```diff
@@ -90,7 +90,7 @@ void showSetAmountDialog (final String qrData) {
             @Override
             public void onClick(DialogInterface dialog, int which) {
                 String amount = edittext.getText().toString();
-                if (amount.equals("")) {
+                if (amount.isEmpty()) {
```

Comment:
Empty amount validation is working fine and why change equals with isEmpty they both accomplish the same thing

Reply:
It is not which is why I opened the issue. Apparently, I don't know why, that works for me. Let me attach a screen shot